pkgs/development/tools/cpm-cmake/default.nix

```diff
@@ -14,6 +14,11 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     hash = "sha256-KIRVztkE72juIgXquZlC6AYo78QKHBD7iCvCa+ri66k=";
   };
 
+  patchPhase = ''
+    substituteInPlace cmake/CPM.cmake \
+      --replace-fail "set(CURRENT_CPM_VERSION 1.0.0-development-version)" "set(CURRENT_CPM_VERSION ${finalAttrs.version})"
```

Comment:
How does upstream handle this? Do they also rewrite the version number in release tarballs?

Reply:
Yep. They have a release artifact that's literally CPM.cmake (the file) with the version set.

While it's theoretically possible we could use that, I feel like building it from source is better, since we need the source anyway to get the documentation.